File: Practice(Beginner)/C++/Valid Triangles.cpp
// Source: https://www.codechef.com/problems/FLOW013

#include <bits/stdc++.h>
using namespace std;

int main() {
    int t, a, b, c;
    cin >> t;
    while(t--){
        cin >> a >> b >> c;
        a+b+c == 180 ? cout << "YES" << endl : cout << "NO" << endl;
    }

    return 0;
}

File: Practice(Beginner)/Python3/Valid_Triangles.py
# Source: https://www.codechef.com/problems/FLOW013

n = int(input())
l = [100,50,10,5,2,1]
for it in range(n):
    a,b,c = map(int,input().split())
    if(a+b+c == 180 and a > 0 and b > 0 and c > 0):
        print("YES")
    else:
        print("NO")
